spec.html

```diff
@@ -32989,13 +32987,11 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( _length_ )</h1>
-        <p>ArrayBuffer called with argument _length_ performs the following steps:</p>
+        <h1>ArrayBuffer ( [ _length_ ] )</h1>
```

Comment:
Function lengths aren't supposed to be about the "typical" number of arguments, they're supposed to be about the number of required arguments. 19.2.4.1 must have simply been overlooked as part of https://github.com/tc39/ecma262/pull/266